document/js-api/index.bs

```diff
@@ -491,23 +491,43 @@ Note: A follow-on streaming API is documented in the <a href="https://webassembl
 <h3 id="modules">Modules</h3>
 
 <pre class="idl">
-enum ImportExportKind {
+enum ExternKind {
   "function",
   "table",
   "memory",
   "global"
 };
 
+// Note: JavaScript objects are not converted into this dictionary type;
+//       it is only used to create JavaScript objects from internal data.
+dictionary AnyType {
```

Comment:
Why can't we use a WebIDL [union type](https://www.w3.org/TR/WebIDL-1/#idl-union) instead?

I suggest naming this type `AnyExternType`.

Reply:
Because two dictionary types are not [distinguishable](https://heycam.github.io/webidl/#dfn-distinguishable) and thus can't be put in a union.

No strong feelings on any of the naming here.